@here/olp-sdk-authentication/README.md

```diff
@@ -2,92 +2,104 @@
- *
- * Client can provide own implementation or can use from @here/olp-sdk-authentication.
- *
- * There are two functions that work for browser and NodeJS.
- * UserAuth can use requestToken() from requestToken.web.ts for browser
- * or requestToken() from requestToken.ts for NodeJS.
- *
- * When a function imports a function using import { requestToken } from "@here/olp-sdk-authentication",
- * the code automatically applies the corresponding function that is applicable to access browser or NodeJS.
- *
- * The following code is applicable both for browser and NodeJS.
- */
-
-import { UserAuth, requestToken } from "@here/olp-sdk-authentication";
-
-const userAuth = new UserAuth({
-    env: "here",
-    credentials: {
-        accessKeyId: "replace-with-your-access-key-id",
-        accessKeySecret: "replace-with-your-access-key-secret"
-    },
-    tokenRequester: requestToken
-});
-```
-
-Get token:
-
-```typescript
-const token: string = await userAuth.getToken();
-```
-
-### Use with Credentials Imported from a File
-
-Download your **credentials.properties** file from the [OLP website](https://developer.here.com/olp/documentation/access-control/user-guide/topics/get-credentials.html).
-Create an instance of UserAuth class and set the path to the file with credentials:
-
-```typescript
-import {
-    UserAuth,
-    requestToken,
-    loadCredentialsFromFile
-} from "@here/olp-sdk-authentication";
-const credentials = loadCredentialsFromFile(
-    "replace-with-your-path-to-credentials.properties"
-);
-const userAuth = new UserAuth({ credentials, tokenRequester: requestToken });
-```
-
-Notice that your credentials.properties file should match the following format:
-
-```txt
-here.user.id = "your-here-user-id"
-here.client.id = "your-here-client-id"
-here.access.key.id = "your-here-access-key-id"
-here.access.key.secret = "your-here-access-key-secret"
-here.token.endpoint.url = "your-here-token-endpoint-url"
-```
-
-Get token:
-
-```typescript
-const token: string = await userAuth.getToken();
-```
+To use the `UserAuth` class with local authorization:
+
+1. Create the `UserAuth` class instance before instantiating any data sources.
+
+    ```typescript
+    /**
+     * The function gets the access token.
+     *
+     * You can provide your own implementation or use one from `@here/olp-sdk-authentication`.
+     *
+     * There are two functions that work for the browser and Node.js.
+     * For the browser, `UserAuth` uses `requestToken()` from requestToken.web.ts.
+     * For Node.js,`UserAuth` uses `requestToken()` from requestToken.ts.
+     *
+     * When a function imports a function using `import { requestToken }` from "@here/olp-sdk-authentication/index.node"`,
+     * the code automatically applies to the corresponding function.
+     *
+     * The following code is applicable only for Node.js.
+     */
+
+    import {
+        UserAuth,
+        requestToken
+    } from "@here/olp-sdk-authentication/index.node";
```

Comment:
I wouldn't use `index.node` at all and follow the schema we already have for e.g. `olp-sdk-fetch`:

```js
{
  "name": "@here/olp-sdk-fetch",
  "version": "1.1.0",
  "description": "Adds a subset of the fetch API for Node.js",
  "main": "index.js",
  "browser": "index.web.js",
```

This is the de-facto standard (index.js for node, index.web.js for browsers) that also works well with webpack and other tools in the ecosystem.

Reply:
Deleted `index.node`.